docs/source/basics/customizing_interface.md

```diff
@@ -148,3 +109,292 @@ Each key in `metadata.parameters` must be the name of one of the parameters of t
+
+```python
+@workflow
+def foo(
+    ...
+):
+    """This line is a short workflow description, displayed in the explore tab and sidebar.
+
+    ...
+    """
+    ...
+```
+
+Example: 
+```python
+@workflow
+def rnaseq(
+    ...
+):
+    """Perform alignment and quantification on Bulk RNA-Sequencing reads.
+
+    ...
+    """
+    ...
+```
+![Short Description](../assets/ui/one-line%20description.png)
+
+### Long Form Description
+
+The body of the workflow function docstring is where you write long-form markdown documentation. This markdown will get rendered in the dedicated workflow "About" tab on your interface. Feel free to include links, lists, code blocks, and more.
+
+```python
+@workflow
+def foo(
+    ...
+):
+    """This line is a short workflow description, displayed in the explore tab
+
+    This line starts the long workflow description in markdown, displayed in
+    this workflow's about tab
+
+    Lists
+    - item1
+    - item2
+    - item3
+
+    ### headers
+
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod 
+    tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, 
+    quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo 
+    consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse 
+    cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat 
+    non proident, sunt in culpa qui officia deserunt mollit anim id est 
+    laborum.
+    """
+    ...
+```
+
+Example: 
+```python
+@workflow 
+def rnaseq(
+    ...
+):
+    """Perform alignment and quantification on Bulk RNA-Sequencing reads
+
+    Bulk RNA-Seq (Alignment and Quantification)
+    ----
+    This workflow will produce gene and transcript counts from bulk RNA-seq
+    sample reads.
+
+    # Workflow Anatomy
+    
+    # Disclaimer
+    
+    This workflow assumes that your sequencing reads were derived from *short-read
+    cDNA seqeuncing* ...
+    
+    # Brief Summary of RNA-seq
+
+    This workflow ingests short-read sequencing files (in FastQ format) that came
+    from the following sequence of steps[^1]:
+      - RNA extraction from sample
+      - cDNA synthesis from extracted RNA
+      - adaptor ligation / library prep
+      - (likely) PCR amplification of library
+      - sequencing of library
+    You will likely end up with one or more FastQ files from this process that hold
+    the sequencing reads in raw text form. This will be the starting point of our
+    workflow.
+    ...
+    """
+```
+
+![Long Description](../assets/ui/long-form%20description.png)
+
+---
+## Adding your workflow to a biological domain on Latch
+
+For public workflows, you may want to classify your workflow to a biological domains to make it easier for future users to discover. 
+
```

Comment:
Think we should add list of common tags already used on the platform. And tell people to make sure they match their tags to tags that already exist on the platform if possible.